WordPress/Classes/ViewRelated/Blog/SiteManagement/SiteTagsViewController.swift

```diff
@@ -9,7 +9,7 @@ final class SiteTagsViewController: UITableViewController {
     }
     private let blog: Blog
 
-    fileprivate let noResultsView = WPNoResultsView()
+    private var noResultsViewController: NoResultsViewController?
```

Comment:
Is there a reason for this to be optional? Could we initialise it here, like we did with the previous view version?

Reply:
Only because I got sucked into the way it was done initially. 😄 It shall be changed!